### ISSUE TITLE
Line Edits

### ISSUE THREAD

**Issue by adammichaelwood** [opendatakit/docs#96]
This could be a perpetual issue, never closed.

Typos sneak in, and branch-based editing makes it inconvenient to go chasing them every time you notice them. So...

IF you are:

 - between issues
 - a little too sleepy in the afternoon to think about high-level stuff
 - a new comer looking for something to do

Start a line-edit branch, fix some typos, and submit a small PR.

## Guidelines for line-edit PRs

 - Typos and basic fixes only; don't rewrite or move sections (You *can* rewrite or move sections, but that needs to be done in individual Issue-Discussion-PR cycles.)
    - Spelling
    - Punctuation
    - Basic grammar
    - Markup corrections
    - Fix links (external and cross references)
 - If your aren't sure, err on the side of not changing something
    - If something needs long task-oriented discussion, file an issue
    - If you need quick clarity on some point of confusion, use the docs-code channel on the [ODK Slack](https://opendatakit.slack.com)
 - Run the build to make sure you didn't introduce errors
 - Don't use line edits to pursue personal grammar pet peeves
 - PR a single document at a time
 - line-edit PRs don't need a long description; the edits should speak for themselves
 - reference this issue number in your PR message, but don't say that it "closes" the issue

**Comment by mish24** [comment]
@adammichaelwood Shall I work on this?

**Comment by adammichaelwood** [comment]
Everyone can work on line edits at any time.
(Don't use the "Closes" syntax in your commit messages.)

**Comment by nadaa** [comment]
Hi,
How can I start with this?
Thanks

**Comment by adammichaelwood** [comment]
@nadaa 

Read the contributor's guide. That provides instruction for setting up your local environment and getting started. Then just pick a file that has spelling or punctuation errors and fix them.

If you need help getting your tooling and authoring environment set up, talk to us on the [developer slack](https://opendatakit.slack.com/), in the #docs-code channel.

**Comment by nadaa** [comment]
Thank you @adammichaelwood.
I am following the contribution guide.  I use windows  (Cygwin) to setup the authoring tools, and I noticed there are no enough instructions in the guide. I will prepare a short doc to describe my approach.

Regards

**Comment by adammichaelwood** [comment]
> I use windows (Cygwin) to setup the authoring tools, and I noticed there are no enough instructions in the guide. 

Yes. We need more details on the Windows experience. Most of us are Mac or Linux people, but we would like to encourage Windows users to contribute.

> I will prepare a short doc to describe my approach.

Great. That would be very helpful.

IF you can incorporate that into the contributors guide, great. IF it makes more sense as a standalone doc, that would be fine too.

There's an issue for this: #95 
We'd love for you to work on that.

**Comment by adammichaelwood** [comment]
Quick note:

**Please do not submit line edits for the Cygwin guide** at this time, unless you are directly working on #159
